src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -54,3 +61,14 @@ const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | Man
         return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
   }
 };
+
+const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
+  const pscData = await getCompanyPsc(req, appData[EntityNumberKey] as string);
```

Comment:
Why "as string"?

Reply:
This was to solve the blocking issue we were discussing on standup, where for some reason we were getting runtime errors if we just used `appData[EntityNumberKey]`, as Typescript was being stupid and saying can't assign string | undefined to string. 
So had to cast as string here.